corporate/finance/check-handling.rst

```diff
@@ -0,0 +1,19 @@
+:og:description: While we prefer electronic payments, RoboJackets occasionally receives paper checks from students or Georgia Tech. Dues and trip fee payments must be recorded in Apiary at the time the check is accepted from the student, and Apiary will send an email receipt to the student.
+
+Check handling policy
+=====================
+
+.. vale Google.Passive = NO
+.. vale Google.We = NO
+.. vale Google.Will = NO
+.. vale write-good.E-Prime = NO
+.. vale write-good.Passive = NO
+
+While we prefer electronic payments, RoboJackets occasionally receives paper checks from students or Georgia Tech.
+Dues and trip fee payments must be `recorded in Apiary <https://my.robojackets.org/docs/officers/payments/accept/#recording-an-offline-payment>`_ at the time the check is accepted from the student, and Apiary will send an email receipt to the student.
+
+Checks should be kept in the safe when not in transit, and deposited within 30 days of receipt.
+
+Checks written out to **Georgia Tech** must be `deposited to our custodial account <https://sofo.gatech.edu/procedures/depositing-funds>`_ by a student officer.
+
+Checks written out to **RoboJackets** must be `deposited to Mercury <https://support.mercury.com/hc/en-us/articles/28768119163156-Depositing-checks>`_ by a corporation or student officer with access to Mercury.
```

Comment:
Written out to or made out to?

Reply:
I think the proper phrasing is "made payable to"?